test/e2e/common/node/container_probe.go

```diff
@@ -217,7 +217,7 @@ var _ = SIGDescribe("Probing container", func() {
 	ginkgo.It("should be restarted with an exec liveness probe with timeout [MinimumKubeletVersion:1.20] [NodeConformance]", func() {
 		// The ExecProbeTimeout feature gate exists to allow backwards-compatibility with pre-1.20 cluster behaviors, where livenessProbe timeouts were ignored
 		// If ExecProbeTimeout feature gate is disabled, timeout enforcement for exec livenessProbes is disabled, so we should skip this test
-		e2eskipper.SkipUnlessExecProbeTimeoutEnabled()
+		e2eskipper.SkipUnlessFeatureGate("ExecProbeTimeout")
```

Comment:
There's a definition of this feature on [pkg/features/kube_features.go#L527](https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L527). I believe it would make sense to use this.